shared/src/main/java/com/couchbase/lite/Function.java

```diff
@@ -0,0 +1,44 @@
+package com.couchbase.lite;
+
+import java.util.Arrays;
+
+public class Function extends Expression {
+
+    private String func = null;
+    private Object param = null;
+
+    private Function(String func, Object param) {
+        this.func = func;
+        this.param = param;
+    }
+
+    public static Function avg(Object expr) {
```

Comment:
expression instead of expr per spec. This will be consistent with https://github.com/couchbase/couchbase-lite-android/blob/feature/2.0/shared/src/main/java/com/couchbase/lite/Expression.java as well.